pkgs/development/compilers/cudatoolkit/hooks/nvcc-setup-hook.sh

```diff
@@ -2,4 +2,4 @@
 
 # CMake's enable_language(CUDA) runs a compiler test and it doesn't account for
 # CUDAToolkit_ROOT. We have to help it locate libcudart
-export NVCC_APPEND_FLAGS+=" -L@cudartRoot@/lib -I@cudartRoot@/include"
+export NVCC_APPEND_FLAGS+=" -L@cudartRootLib@/lib -L@cudartRootStatic@/lib -I@cudartRootInclude@/include"
```

Comment:
Maybe just cudartLib, cudartStatic. I only named it root because I thought about a merged "prefix" directory

Reply:
Fixed in 96d6b7ce1c949e00d214dd9ce7e189fa663c63c8 -- although, I am curious... is there a standard for naming environment variables which are meant to be internal to the derivation?